addons/context_actions/functions/fnc_toggleCaptive.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+/*
+ * Author: Eclipser
+ * Toggles ACE captive states of units
```

Comment:
Missing period at the end. Maybe; "Toggles the ACE captive state of the given units."